gems/smithy-client/sig/smithy-client/http/request.rbs

```diff
@@ -0,0 +1,18 @@
+module Smithy
+  module Client
+    module HTTP
+      class Request
+
+        def initialize: (?Hash[Symbol, untyped]) -> void
+
+        attr_reader endpoint: endpoint_url?
+        attr_accessor http_method: String
+        attr_reader headers: Headers
+        attr_reader body:  _ReadableIO?
```

Comment:
since we have stringio defined in our rbs collection - we could use it here

Reply:
The ReadbleIO interface is actually I think more correct here since we do allow any "io-like" object that supports read.